### PR TITLE
Windup-3431: Rebranding to MTR - CLI Guide

### PR DESCRIPTION
**Pull request by sbeskin-redhat**
Windup-3431: Rebranding to MTR - CLI Guide
Resolves: https://issues.redhat.com/browse/WINDUP-3431
MTR 1.0

### CODE REVIEW

**Review by sshveta — APPROVED** [review]

File: docs/topics/templates/document-attributes.adoc [hunk 1]
```diff
@@ -98,6 +98,7 @@ endif::[]
 // URLs for downloads on developers.redhat.com
 :MTADownloadPageURL: https://developers.redhat.com/products/mta/download
```
Comment (sshveta) [inline docs/topics/templates/document-attributes.adoc:99]: This URL needs to be updated.

**Review by sshveta — CHANGES_REQUESTED** [review]

File: docs/topics/what-is-the-toolkit.adoc [hunk 2]
```diff
@@ -23,6 +23,10 @@ The {ProductName} ({ProductShortName}) is an extensible and customizable rule-ba
 * Containerizing applications and making them cloud-ready
 * Migrating from Java Spring Boot to Quarkus
 * Updating from Oracle JDK to OpenJDK
+* Upgrading from OpenJDK 8 to OpenJDK 11
+* Upgrading from OpenJDK11 to OpenJDK 17
+* Migrating EAP Java applicatons to Azure
+* Migrating Spring Boot Java applications to Azure
 
 For more information about use cases and migration paths, see the link:https://developers.redhat.com/products/mta/use-cases[MTA for developers] web page.
```
Comment (stillalearner) [inline docs/topics/what-is-the-toolkit.adoc:31]: Replace MTA with MTR

File: docs/topics/add-rule-to-mta.adoc [hunk 3]
```diff
@@ -6,11 +6,11 @@
 [id="add-rule-to-mta_{context}"]
```
Comment (stillalearner) [inline docs/topics/add-rule-to-mta.adoc:6]: mta to mtr

File: docs/topics/exclude-files-and-packages.adoc [hunk 4]
```diff
@@ -36,8 +36,10 @@ These files contain a regex string detailing the name to exclude, with one file
 {ProductShortName} searches the following locations:
 
 * `~/.mta/ignore/`
```
Comment (stillalearner) [inline docs/topics/exclude-files-and-packages.adoc:38]: mta to mtr

File: docs/topics/installing-intellij-idea-plugin.adoc [hunk 5]
```diff
@@ -11,7 +11,7 @@ You can install the {ProductShortName} plugin in the Ultimate and the Community
 .Prerequisites
 include::snippet_jdk-hardware-mac-prerequisites.adoc[]
 
-* The latest version of `mta-cli` from the link:{MTADownloadPageURL}[{ProductShortName} download page]
+* The latest version of `windup-cli` from the link:{MTADownloadPageURL}[{ProductShortName} download page]
```
Comment (stillalearner) [inline docs/topics/installing-intellij-idea-plugin.adoc:14]: MTA to MTR

File: docs/topics/installing-web-console-on-openshift.adoc [hunk 6]
```diff
@@ -41,7 +41,7 @@ ifdef::ocp-42[]
 .Procedure
 
 . Navigate to the link:{MTADownloadPageURL}[{ProductShortName} Download page] and download the {WebName} `Local install & OpenShift` file.
```
Comment (stillalearner) [inline docs/topics/installing-web-console-on-openshift.adoc:43]: MTA to MTR

File: docs/topics/cli-run.adoc [hunk 7]
```diff
@@ -10,12 +10,12 @@ You can run {ProductShortName} against your application.
 
 .Procedure
 
-. Open a terminal and navigate to the `<MTA_HOME>/bin/` directory.
-. Execute the `mta-cli` script, or `mta-cli.bat` for Windows, and specify the appropriate arguments:
+. Open a terminal and navigate to the `<{ProductShortName}_HOME>/bin/` directory.
+. Execute the `windup-cli` script, or `windup-cli.bat` for Windows, and specify the appropriate arguments:
```
Comment (stillalearner) [inline docs/topics/cli-run.adoc:14]: mtr-cli

File: docs/topics/cli-run.adoc [hunk 8]
```diff
@@ -10,12 +10,12 @@ You can run {ProductShortName} against your application.
...
 +
 [source,terminal]
 ----
-$ ./mta-cli --input /path/to/jee-example-app-1.0.0.ear \
+$ ./windup-cli --input /path/to/jee-example-app-1.0.0.ear \
```
Comment (stillalearner) [inline docs/topics/cli-run.adoc:18]: mtr-cli

File: docs/topics/web-start.adoc [hunk 9]
```diff
@@ -8,11 +8,11 @@
 
 Run the script to start the {WebName}.
 
-[source,options="nowrap",subs="+quotes"]
+[source,options="nowrap",subs="attributes+"]
 ----
 $ <{ProductShortName}_HOME>/run_mta.sh
 ----
 
 NOTE: In a Windows environment, use the `run_mta.bat` script.
```
Comment (sshveta) [inline docs/topics/web-start.adoc:16]: script name is not run_mta.sh . 
It is run_windup.bat

File: docs/rules-development-guide/master.adoc [hunk 10]
```diff
@@ -21,7 +21,7 @@ include::topics/rules-guide-intro.adoc[leveloffset=+2]
 // Use of <{ProductShortName}_HOME> in this Guide
 include::topics/about-home-var.adoc[leveloffset=+3]
 
-// {ProductShortName} Rules
+// MTA Rules
```
Comment (sshveta) [inline docs/rules-development-guide/master.adoc:24]: Replace MTA with MTR

File: docs/rules-development-guide/master.adoc [hunk 11]
```diff
@@ -89,7 +89,7 @@ For more information on categorizing tasks, see xref:rule-categories_{context}[U
 
 === Additional resources
 
-// Review the Existing {ProductShortName} XML Rules
+// Review the Existing MTA XML Rules
```
Comment (sshveta) [inline docs/rules-development-guide/master.adoc:92]: Replace MTA with MTR

File: docs/topics/cli-args.adoc [hunk 12]
```diff
@@ -154,7 +154,7 @@ The `--source` argument now provides version support, which follows the link:htt
 ====
 When migrating to JBoss EAP, be sure to specify the version, for example, `eap:6`. Specifying only `eap` will run rulesets for all versions of JBoss EAP, including those not relevant to your migration path.
 
-See link:{ProductDocIntroTo{ProductShortName}GuideURL}/index#migration_paths_getting-started-guide[Supported migration paths] in _{IntroToMTABookName}_ for the appropriate JBoss EAP version.
+See link:{ProductDocIntroToMTAGuideURL}/index#migration_paths_getting-started-guide[Supported migration paths] in _{IntroToMTABookName}_ for the appropriate JBoss EAP version.
```
Comment (sshveta) [inline docs/topics/cli-args.adoc:157]: Replace MTA with MTR

File: docs/topics/cli-args.adoc [hunk 13]
```diff
@@ -175,7 +175,7 @@ The `--target` argument now provides version support, which follows the link:htt
 ====
 When migrating to JBoss EAP, be sure to specify the version in the target, for example, `eap:6`. Specifying only `eap` will run rulesets for all versions of JBoss EAP, including those not relevant to your migration path.
 
-See link:{ProductDocIntroTo{ProductShortName}GuideURL}/index#migration_paths_getting-started-guide[Supported migration paths] in _{IntroToMTABookName}_ for the appropriate JBoss EAP version.
+See link:{ProductDocIntroToMTAGuideURL}/index#migration_paths_getting-started-guide[Supported migration paths] in _{IntroToMTABookName}_ for the appropriate JBoss EAP version.
```
Comment (sshveta) [inline docs/topics/cli-args.adoc:178]: Replace MTA with MTR

File: docs/topics/installing-web-console-or-cli-tool.adoc [hunk 14]
```diff
@@ -51,13 +51,13 @@ ifdef::web-console-guide[]
...
 ----
 
 * Windows operating system:
 +
 ----
-C:\<{ProductShortName}_HOME>\run_{LC_PSN}.bat
+C:\<MTR_HOME>\run_mtr.bat
 ----
 
 . Open a browser and navigate to `\http://localhost:8080/mta-web`. The {WebName} login page is displayed in your browser.
```
Comment (sshveta) [inline docs/topics/installing-web-console-or-cli-tool.adoc:63]: Replace MTA with MTR

File: docs/topics/vs-code-extension-run-configuration.adoc [hunk 15]
```diff
@@ -10,7 +10,7 @@ You can configure multiple run configurations to run against each project you im
 
 .Prerequisites
 
-* `mta-cli` executable installed. You can download the `mta-cli` executable from link:https://developers.redhat.com/products/mta/download[{ProductShortName} download].
+* `windup-cli` executable installed. You can download the `windup-cli` executable from link:https://developers.redhat.com/products/mta/download[MTA download].
```
Comment (sshveta) [inline docs/topics/vs-code-extension-run-configuration.adoc:13]: Replace MTA with MTR

**Review by sshveta — CHANGES_REQUESTED** [review]

File: docs/topics/web-start.adoc [hunk 16]
```diff
@@ -8,11 +8,11 @@
 
 Run the script to start the {WebName}.
 
-[source,options="nowrap",subs="+quotes"]
+[source,options="nowrap",subs="attributes+"]
 ----
 $ <{ProductShortName}_HOME>/run_mta.sh
```
Comment (stillalearner) [inline docs/topics/web-start.adoc:13]: run_windup.sh

File: docs/topics/installing-web-console-or-cli-tool.adoc [hunk 17]
```diff
@@ -51,13 +51,13 @@ ifdef::web-console-guide[]
 * Linux operating system:
 +
 ----
-$ <{ProductShortName}_HOME>/run_{LC_PSN}.sh
+$ <MTR_HOME>/run_mtr.sh
```
Comment (stillalearner) [inline docs/topics/installing-web-console-or-cli-tool.adoc:54]: @sshveta it should be run_windup.sh right?

File: docs/topics/installing-web-console-or-cli-tool.adoc [hunk 18]
```diff
@@ -51,13 +51,13 @@ ifdef::web-console-guide[]
...
 ----
 
 * Windows operating system:
 +
 ----
-C:\<{ProductShortName}_HOME>\run_{LC_PSN}.bat
+C:\<MTR_HOME>\run_mtr.bat
```
Comment (stillalearner) [inline docs/topics/installing-web-console-or-cli-tool.adoc:60]: @sshveta here also, run_windup.bat  ?

**Pull request merged**